### PR TITLE
Post app GET and Post Request

### PR DESCRIPTION
**Pull request by berfinsimsekk**
The app provides support for GET and POST requests and a simple user interface. The code for the app needs to be reviewed by 2 people before merging as discussed in meeting 15.

### CODE REVIEW

File: practice_app/apps/post/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    operations = [
+        migrations.CreateModel(
+            name='Post',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.CharField(max_length=500)),
+                ('description', models.CharField(max_length=2000)),
+                ('post_type_label', models.CharField(max_length=50)),
+                ('nsfw', models.BooleanField()),
```
Comment (haticerk) [inline practice_app/apps/post/migrations/0001_initial.py:21]: I could not see any info about nsfw, maybe you can delete from there.

File: practice_app/apps/post/templates/postindex.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  <label for="id">Post ID:</label><br>
+  <input type="text" id="id" name="id" required><br>
+ <button type="submit" style="margin-top: 10px;"  class="btn btn-success">Get Post Info</button>
+</form>
+    <form id="post" class="form-group" method="POST" action="{%url 'createPost' %}" >
+        {% csrf_token %}
+  <label for="id">Create New Post:</label><br>
+        <input type="text"style="margin-top: 1px;" placeholder="title" id="title" name="title" required><br>
+        <input type="text"style="margin-top: 1px;"  placeholder="description" id="description" name="description" required><br>
+        <input type="text"style="margin-top: 1px;"  placeholder="post type label" id="post_type_label" name="post_type_label" required><br>
```
Comment (haticerk) [inline practice_app/apps/post/templates/postindex.html:49]: Maybe, you can write placeholder's strating with capital letter to a little bit better UI.

**Review by haticerk — APPROVED** [review]
I think everything is okay and works correctly. Good job!

**Review by LehabuL — APPROVED** [review]
It works without a problem. The only issue is if you don't provide the fields when creating a post it asks the user to fill it, however once you type space it says the post is created but you can't see it from Get Post Info. I think this is really an edge case so we can merge it for now.

File: practice_app/apps/post/templates/postindex.html (hunk 2 above)
Comment (berfinsimsekk) [reply practice_app/apps/post/templates/postindex.html]: Thanks, I have updated the html file.

**Pull request merged**